pkg/targetgroupbinding/resource_manager.go

```diff
@@ -382,6 +382,10 @@ func (m *defaultResourceManager) registerPodEndpoints(ctx context.Context, tgARN
 	if err != nil {
 		return err
 	}
+	tgInfo, err := m.targetsManager.DescribeTargetGroup(ctx, tgARN)
```

Comment:
I'd prefer caching the target group vpc Id so we don't incur the DescribeTargetGroup call with every register.

Reply:
+1, we should minimize DescribeTargetGroup calls in this critical path.

we can either add cache or mutate the targetGroupBinding spec during MutationWebhook to capture vpcID in spec/annotation.

I prefer to have a cache for now given its simper and will be a two way door.

Reply:
Thank you for your advice!
I agree that. But I'm afraid I'm not sure how to best implement a cache for the controller.

Could you give me some reference information (e.g. similar implement, sample code)?